api/core/agents/agent_guidelines.py

```diff
@@ -0,0 +1,21 @@
+# Instruction Templating specifics, this part is separated
+INSTRUCTION_TEMPLATING_GUIDELINES = """
+- For short fields (ex: user name, user email, etc.) you can 'inline' the variable directly in the instructions sentences:
+{% raw %}
+- The user name is {{ user.name }}
+{% endraw %}
+
+- For longer fields (ex: email content), you can use the following syntax (preferably at the end of the instructions):
+{% raw %}
+- The email content to process is:
+{{ user.email_content }}
+{% endraw %}
+
+- For deeply nested fields, you do not need to specify the path to every single field, you can just inject the parent object where it makes sense:
+{% raw %}
+The issues to generate the release note from are:
+{{ issues }}
```

Comment:
Do we have an idea of what the rendering will look like ? 
Do we need to have a custom format for plain objects ? maybe yaml ?

Reply:
right now the injected object is a "json" string.